homeassistant/components/fireservicerota/switch.py

```diff
@@ -140,10 +141,11 @@ async def async_update(self) -> bool:
         data = await self._client.async_response_update()
 
         if not data or "status" not in data:
-            return
+            return False
```

Comment:
`async_update` is a standard Entity method and it should not return a boolean. It should return None.

Reply:
Now I see we call `self.async_schedule_update_ha_state(True)` above. That will call this method. It's a legacy pattern.